packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx

```diff
@@ -79,8 +79,8 @@ const DateFilterInputs: FC<FilterInputsProps<DateFilterRule>> = (props) => {
                                         ...filterRule,
                                         values: [
                                             moment(value)
-                                                .utc(true)
-                                                .startOf('year'),
+                                                .startOf('year')
```

Comment:
Same as above, this is triggered when you select a month from the dropdown, when I click on July, it selects June.